src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/Microsoft.DotNet.UpgradeAssistant.Abstractions.csproj

```diff
@@ -46,4 +46,8 @@
       <Version>5.0.0</Version>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <None Update="build\*" Pack="true" PackagePath="" />
```

Comment:
this does not appear to support the goal of updating and reorganizing documentation

Reply:
It's part of creating a more seamless dev experience for extensions. I could separate it out, but figured I'd shoe-horn it into here...